sdk/src/main/java/io/dapr/client/domain/LockRequest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client.domain;
+
+
+/**
+ * A request to lock.
+ */
+public class LockRequest {
```

Comment:
Make this class json serializable with only the required properties ie resourcrId etc being in the json form ie except storeName.

Reply:
I think this is a class open to users. If `storeName` is excluded, the meaning of LockRequest will be incomplete. It can be compared to `GetBulkStateRequest`, `DeleteStateRequest`, etc. What do you think?

Reply:
I think it is ok to have `storeName` excluded in the JSON format with correct documentation. @artursouza WDYT?

Reply:
I agree. We can annotate to make it not part of the output.

Reply:
Done.